docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,727 @@
+    +--------------------------+------------------+---------------+--------------------------+--------+
+    | 5f44be31-0588-4f33-883d- | vpn-conn-to-kna1 | 203.0.113.101 | psk                      | ACTIVE |
+    | 9e2d97be55e1             |                  |               |                          |        |
+    +--------------------------+------------------+---------------+--------------------------+--------+
+    ```
+    
+    If you want more information regarding a specific connection, type 
+    something like this:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1
+    ```
+    
+    ```plain
+    +--------------------------+----------------------------------------------------+
+    | Field                    | Value                                              |
+    +--------------------------+----------------------------------------------------+
+    | Authentication Algorithm | psk                                                |
+    | Description              |                                                    |
+    | ID                       | 5f44be31-0588-4f33-883d-9e2d97be55e1               |
+    | IKE Policy               | 5782b141-afcc-4327-9ac5-b8cd2e110c6a               |
+    | IPSec Policy             | 15ec761f-1642-49b6-b5a2-e43624c5752d               |
+    | Initiator                | bi-directional                                     |
+    | Local Endpoint Group ID  | 51895e0e-fa3a-43d3-8037-5eea073fb77f               |
+    | Local ID                 |                                                    |
+    | MTU                      | 1500                                               |
+    | Name                     | vpn-conn-to-kna1                                   |
+    | Peer Address             | 203.0.113.101                                      |
+    | Peer CIDRs               |                                                    |
+    | Peer Endpoint Group ID   | a96bb9ef-d0ec-4174-93ae-a8e655910f94               |
+    | Peer ID                  | 203.0.113.101                                      |
+    | Pre-shared Key           | de12db260ee1b9c0b9e624d910c9a9dbddec13dc24d60332   |
+    | Project                  | dfc700467396428bacba4376e72cc3e9                   |
+    | Route Mode               | static                                             |
+    | State                    | True                                               |
+    | Status                   | ACTIVE                                             |
+    | VPN Service              | d74d51f0-182d-4d88-952a-1d593ce696fd               |
+    | dpd                      | {'action': 'hold', 'interval': 30, 'timeout': 120} |
+    | project_id               | dfc700467396428bacba4376e72cc3e9                   |
+    +--------------------------+----------------------------------------------------+
+    ```
+
+## Testing the site-to-site VPN connection
```

Comment:
Now that there is a section on enabling and disabling a connection, I would move this entire section down to the very end (because it also serves as a test of whether the connection has been successfully disabled and re-enabled).